noir-projects/aztec-nr/aztec/src/utils/bytes.nr

```diff
@@ -1,9 +1,17 @@
+// This file contains some unused functions that may be needed later (e.g. those filling top 6 bits of a 31-byte
+// field). Little endian functions are kept for reference but likely won't be used. Unused functions should be
+// removed before audit.
```

Comment:
Included this comment based on [this discussion](https://aztecprotocol.slack.com/archives/C07R3GTKJ49/p1741377330589099) on slack.